content/paths/transfer__put_users_id_folders_0.yml

```diff
@@ -41,16 +41,16 @@ requestBody:
       schema:
         type: object
         required:
-          - enterprise
+          - owned_by
         properties:
-          enterprise:
+          owned_by:
             type: object
             description: |-
               The user who the folder will be transferred to
             required:
-              - id
+              - user_id
```

Comment:
Actually, I am not sure this should be `owned_by.user_id`. First off the ticket says `owned_by.userid`, and secondly we use `owned_by.id` in other places. Would be worth validating.